railties/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Rails 5.0.0.beta1 (December 18, 2015) ##
 
+*   Fix using `add_source` with a block after using `gem` in a custom generator.
+
+    *Will Fisher*
```

Comment:
Can you move that above the ` ## Rails 5.0.0.beta1 (December 18, 2015) ##` line?